annotation-client/src/test/java/eu/europeana/annotation/client/integration/webanno/tag/BodyTypesTest.java

```diff
@@ -0,0 +1,354 @@
+		searchByBodyValue(VALUE_ID+"\""+storedAnno.getAnnotationId().getIdentifier()+"\"", TOTAL_BY_ID_FOUND);
+		searchByBodyValue(VALUE_DESCRIBING_BODY_VALUE, TOTAL_BY_ID_FOUND);
+		searchByBodyValue(VALUE_DESCRIBING_TARGET_SCOPE, TOTAL_BY_ID_FOUND);
+		
+		// validate fields
+		assertTrue(storedAnno.getMotivation().equals(MotivationTypes.DESCRIBING.name().toLowerCase()));
+		assertEquals(storedAnno.getBody().getInternalType(), BodyInternalTypes.TEXT.name());
+		TextBody textBody = ((TextBody) storedAnno.getBody());
+		assertNotNull(textBody.getValue());
+		assertTrue(textBody.getValue().equals("... this is the textual description of the item ..."));
+		Target target = storedAnno.getTarget();
+		assertNotNull(target.getSource());
+		assertTrue(target.getSource().equals("http://www.europeana1914-1918.eu/attachments/2020601/20841.235882.full.jpg"));
+		assertNotNull(target.getScope());
+		assertTrue(target.getScope().equals("http://data.europeana.eu/item/07931/diglit_uah_m1"));
+
+		// remove tag
+		deleteAnnotation(storedAnno);
+	}
+	
+	@Test
+	public void createLink() throws IOException, JsonParseException, IllegalAccessException, IllegalArgumentException, InvocationTargetException {
+		
+		String requestBody = getJsonStringInput(LINK_STANDARD);
+		
+		// create indexed tag
+		Annotation storedAnno = createTag(requestBody);
+		
+		// search for indexed id and textual values
+		searchByBodyValue(VALUE_ID+"\""+storedAnno.getAnnotationId().getIdentifier()+"\"", TOTAL_BY_ID_FOUND);
+		searchByBodyValue(VALUE_TARGET_LINK, TOTAL_BY_ID_FOUND);
+		
+		// validate fields
+		assertTrue(storedAnno.getMotivation().equals(MotivationTypes.LINKING.name().toLowerCase()));
+		Target target = storedAnno.getTarget();
+		assertNotNull(target.getValues());
+		assertTrue(target.getValues().contains("http://data.europeana.eu/item/123/xyz"));
+
+		// remove tag
+		deleteAnnotation(storedAnno);
+	}
+	
+	@Test
+	public void createSemanticLink() throws IOException, JsonParseException, IllegalAccessException, IllegalArgumentException, InvocationTargetException {
+		
+		String requestBody = getJsonStringInput(LINK_SEMANTIC);
+		
+		// create indexed tag
+		Annotation storedAnno = createTag(requestBody);
+		
+		// search for indexed id and textual values
+		searchByBodyValue(VALUE_ID+"\""+storedAnno.getAnnotationId().getIdentifier()+"\"", TOTAL_BY_ID_FOUND);
+		searchByBodyValue(VALUE_TARGET_LINK_SEMANTIC, TOTAL_BY_ID_FOUND);
+		
+		// validate fields
+		assertTrue(storedAnno.getMotivation().equals(MotivationTypes.LINKING.name().toLowerCase()));
+		Target target = storedAnno.getTarget();
+		assertNotNull(target.getHttpUri());
+		assertTrue(target.getHttpUri().equals("http://data.europeana.eu/item/2059207/data_sounds_T471_5"));
+
+		// remove tag
+		deleteAnnotation(storedAnno);
+	}
+	
+	@Test
+	public void createGraph() throws IOException, JsonParseException, IllegalAccessException, IllegalArgumentException, InvocationTargetException {
```

Comment:
this is a search not a create test, please rename

Reply:
changed to search...